keras_cv/backend/ops.py

```diff
@@ -11,11 +11,16 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from keras_cv.backend.config import detect_if_tensorflow_uses_keras_3
 from keras_cv.backend.config import multi_backend
 
-if multi_backend():
+if detect_if_tensorflow_uses_keras_3():
```

Comment:
I don't get how this could work. If tensorflow is using keras 3 (and keras 2 is properly uninstalled), won't `keras.preprocessing` not exist here?

Reply:
Updated